### ISSUE TITLE
Messages should be announced via aria-live

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/nextcloud-dialogs#621]
Ref https://github.com/nextcloud/spreed/issues/7105

> New chat messages should be announced via `aria-live="polite"`, including author and message content, see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

Reading:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

> aria-live="assertive" should only be used for time-sensitive/critical notifications that absolutely require the user's immediate attention. Generally, a change to an assertive live region will interrupt any announcement a screen reader is currently making. As such, it can be extremely annoying and disruptive and should only be used sparingly.

My suggestion would be to either use `aria-live="assertive"` for all, or at least the elements with `aria-live="polite"`

As far as I can see toastify-js does not allow this. So either we need to deal with it upstream or have to look for another library

**Comment by nickvergessen** [comment]
Moved to https://github.com/nextcloud/nextcloud-dialogs/issues/255
